CMakeLists.txt

```diff
@@ -875,7 +875,11 @@ if(WITH_OPTIM)
         endif()
         if(WITH_CRC32_VX)
             check_vgfma_intrinsics()
-            if(HAVE_VGFMA_INTRIN)
+            if(HAVE_VGFMA_INTRIN AND
+                NOT ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang" AND
+                "${CMAKE_C_COMPILER_VERSION}" VERSION_GREATER_EQUAL 18 AND
```

Comment:
``VERSION_GREATER_EQUAL`` - cmake 3.7+, but zlib-ng required cmake 3.5+.

Reply:
i guess bumping the version is not an option. so i'll split it into two checks.